docs/REST API/rest-api.md

```diff
@@ -2531,6 +2513,198 @@ Returns the information of the host specified using the `uuid`, `osquery_host_id
 
 > Note: `installed_paths` may be blank depending on installer package. For example, on Linux, RPM-installed packages do not provide installed path information.
 
+#### Get host by device token
+
+Returns information about the host specified by `token`. This is the API route used by the **My device** page in Fleet desktop to display information about the host to the end user.
```

Comment:
Just occurred to me since the last time I reviewed: this has a different URL structure than the other host endpoints— does the data it returns differ from "Get host" or "Get host by identifier"? 

Since all 3 of the "get host" endpoints have really long example responses that are difficult to compare, I think we should clarify here whether this one includes information the other endpoints don't have (or the other way around).

Reply:
> does the data it returns differ from "Get host" or "Get host by identifier"?

Yes, it does. Good point that we should clarify. I added this.